src/client/AM.PA.MonitoringTool/SlackTracker/Analysis/Helpers.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using SlackTracker.Data;
+using SlackTracker.Data.SlackModel;
+using OpenNLP.Tools.Tokenize;
+using OpenNLP.Tools.SentenceDetect;
+using OpenNLP.Tools.PosTagger;
+using SlackTracker.Analysis;
+using System.Linq;
+using Shared;
+
+namespace SlackTracker.Analysis
+{
+    class Helpers
+    {
+        #region Text Processing
+        /// <summary>
+        /// Gets Parts of Speech tags for words in document
+        /// </summary>
+        /// <param name="tokens"></param>
+        /// <returns>A list of POS tags</returns>
+        public static List<string> getPosTags(List<string> tokens)
+        {
+            List<string> tags = new List<string>();
+
+            var _modelPath = AppDomain.CurrentDomain.BaseDirectory + "../../../SlackTracker/Analysis/resources/models/";
```

Comment:
I don't think adding models as resource will work since the OpenNLP API takes the model path(string) as input. i.e the line below "new EnglishMaximumEntropyPosTagger(_modelPath + "EnglishPOS.nbin", _modelPath + @"Parser\tagdict");"